packages/sui-studio/scripts/prepare-only-changes.js

```diff
@@ -12,31 +11,41 @@ const {NO_COMPONENTS_MESSAGE} = require('../config')
 
 const log = console.log
 const error = console.error
-const {exec} = require('child_process')
 const fs = require('fs-extra')
+const path = require('path')
+const fetch = require('node-fetch')
+const gitUrlParse = require('git-url-parse')
 
-function getComponentsList(stdout) {
-  const COMMIT_MESSAGE_PATTERN = /(?<context>[a-zA-Z]+)\/(?<component>[a-zA-Z]+)/
+const MAX_GITHUB_API_RESULTS = 100
+const PUBLIC_GITHUB_HOST = 'github.com'
+const PUBLIC_GITHUB_API_URL_PATTERN =
+  'https://api.github.com/repos/:org/:repo/pulls/:pull_request/commits?per_page=:results'
```

Comment:
from github's docs, it looks like it's possible to get the whole, non-paginated list of commits for a PR 

https://docs.github.com/en/rest/reference/repos#list-commits

Reply:
I know it's a bit of an edge case to have more than 100 commits per PR... but for those `PRs` that leverage `commit-all` it may be useful...